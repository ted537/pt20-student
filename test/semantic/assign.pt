using output;
var x : file like 2;
assign("myfile.txt",x);
