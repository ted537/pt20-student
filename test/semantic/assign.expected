.tFileDescriptor .tLiteralInteger .tFileBind .tLiteralAddress .tStoreInteger .tFileDescriptor .tLiteralInteger .tFileBind .tLiteralAddress .tStoreInteger .tTrapBegin .tLiteralAddress .tVarParm .tParmEnd .tSkipString .tStringData .tLiteralAddress .tFetchChar .tParmEnd .tLiteralInteger .tParmEnd .tTrap .tTrapBegin .tTrap
